Dizzy/Screens/Discovery/Cells/DiscoveryPlaceCell.swift

```diff
@@ -67,12 +70,17 @@ class DiscoveryPlaceCell: UICollectionViewCell {
         backgroundColor = .white
         setupStackView()
         setupLabels()
+        placeImageView.layer.cornerRadius = placeImageViewSize/2
```

Comment:
maybe better to put this part of the code in a function.

Reply:
done :)